test/browser_tests/spec_suites/large_screen/large_contact-us.js

```diff
@@ -1,24 +0,0 @@
-'use strict';
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/0cc8707a3083e1ff9e721e0a21a78aa4ad8cf2f5